content/operator/ltg/index.de.md

```diff
@@ -0,0 +1,94 @@
+
+{{% highlight tip %}}
+FIP 50 Tickets und Freifahrtscheine von LTG-Link, gelten auch in Zügen von LTG-Link auf der Strecke Vilnius – Riga. [^1]
+{{% /highlight %}}
+
+## Zugkategorien und Reservierungen
+
+In Litauen wird nicht zwischen verschiedenen Zugkategorien unterschieden.
+
+{{% highlight important %}}
+LTG-Link verlangt in Fernverkehrszügen eine verpflichtende kostenlose Sitzplatzreservierung (eine Auflistung der Verbindungen ist auf [LTG-Link Website](https://ltglink.lt/en/passengers-with-interrail-eurail-passes) zu finden). Mit FIP Freifahrtschein muss jedoch auch in Zügen ohne Reservierungspflicht ein kostenloses Ticket (_0er-Fahrschein_) vor der Fahrt erworben werden.
+
+Züge mit Reservierungspflicht sind teilweise schon viele Tage im Voraus ausgebucht. In der Hochsaison und an Feiertagen zwischen Vilnius und Klaipeda bzw. Riga sind Züge teilweise 2-3 Wochen zuvor ausgebucht.
+
+Buchungsbeginn ist bei Inlandsfahrkarten 30 Tage vor Abfahrt und bei Auslandsfahrten 60 Tage vor Abfahrt.
+
+Somit ist vor **jeder** Fahrt mit FIP Freifahrtschein ein kostenloses Ticket (0er-Fahrschein) mit/ohne Pflichtreservierung oder ein FIP 50 Ticket zu erwerben, das ggf. eine kostenlose Reservierung enthält. (siehe [Ticket- und Reservierungskauf](#ticket--und-reservierungskauf)).
+{{% /highlight %}}
+
+FIP Vergünstigungen der 1. Klasse gelten in allen Klassen. FIP Vergünstigungen der 2. Klasse gelten in der 2. und 3. Klasse.
+
+Fahrgäste der 1. Klasse erhalten in Fernverkehrszügen kostenlose Getränke und einen kleinen Imbiss.
+
+Die Züge führen Wagen der 1., 2. und 3. Klasse. Mit Ausnahme einiger Saisonzüge zwischen Vilnius und Klaipeda (mit allen Klassen) weisen die Züge entweder die Kombination 1./2. oder 2./3. Klasse auf.
+
+## Ticket- und Reservierungskauf
+
+### Online
+
+{{% booking id="ltg-website" /%}}
+
+### Vor Ort
+
+Reisezentren befinden sich nur noch in Vilnius, Kaunas und Klaipeda; ob sie 2026 noch Fahrkarten verkaufen oder reine Infoschalter werden, war Ende 2025 noch nicht absehbar.
+
+Die inzwischen an vielen Stationen aufgestellten Fahrkartenautomaten geben keine FIP-Fahrscheine aus (Stand: 11/2025).
+
+### Im Zug
+
+FIP-ermäßigte Fahrkarten können nur im Zug gekauft werden, wenn die Abfahrt von einem Bahnhof erfolgt, an dem der Fahrkartenschalter geschlossen und/oder der Automat defekt ist.
+
+## Ermäßigungen
+
+Mit regulären Fahrkarten reisen maximal 2 Kinder bis einschließlich 7 Jahre in der 2. Klasse kostenlos, müssen dann aber auf dem Schoß der Eltern sitzen. Wenn die Kinder bis 7 Jahre einen eigenen Sitz erhalten sollen, sind 50% vom Erwachsenentarif zu zahlen. Kinder bis 18 Jahre (oder als Schüler:innen noch bis 23 Jahre) erhalten 50% Ermäßigung auf den Erwachsenentarif. Personen ab 18 Jahren zahlen den vollen Erwachsenentarif, jedoch gibt es zahlreiche Ermäßigungen für Studierende, Senior:innen ab 80 Jahren, Soldat:innen, Chemotherapie-Patient:innen und Gruppen ab 3 Personen.
```

Comment:
Und in der 1. Klasse zahlen Kinder immer 50%, auch wenn die auf dem Schoß der Eltern sitzen?

Reply:
Ich weiß es nicht :D

Reply:
Ok, das können wir im Nachgang klären :)